### DISCUSSION TITLE
Finish Demo Version

### DISCUSSION THREAD

**Discussion by PortMaster1** [JD-Productions/Quest-Island#5] [General]
## Finish Demo Version
I think we should finish the demo version of Quest Island. Then we can open a new branch for development on the main game.

**Comment by PortMaster1** [comment]
[ I will refer to the regular version of the game as the __*main*__ game. ]

I think we should exclude the dialogue from the demo, because it has a lot to do with unavailable features.
What do you think, @park102?

**Reply by park102** [reply]
Sounds good
